### DISCUSSION TITLE
Pre-release version with custom templates

### DISCUSSION THREAD

**Discussion by fredguth** [quarto-dev/quarto-cli#3862] [Q&A] [answered]
I decided to download the 1.3.83 pre-release version and noticed that a template I created (`fredguth/tufte-quarto`) is not working properly in this version.  

This leads me to question:
- **What is the version strategy of quarto and when should I think about updating my templates to a new version?**

I couldn't find anything specific about templates in the pre-release documentation (https://quarto.org/docs/prerelease/1.3.html)

**Comment by dragonstyle** [comment, marked as answer]
In general, we're doing our best to remain compatible in the current releases. This will get tested particularly in places like CSS selectors, but it is definitely worth raising issues when you encounter breakage merely from changing versions. 

Can you say more about the issue that you've run into - I'm happy to help your troubleshoot it.

**Reply by fredguth** [reply]
I created a new project with `quarto use template fredguth/tufte-quarto`.

1) `quarto preview`: has some css differences but it is ok
2) `quarto render --to pdf` yields `ERROR: The referenced format resource 'tufte-book.cls' does not exist.`
Despite `tufte-book.cls` being  in _extension.yml `format-resources`

**Reply by dragonstyle** [reply]
I can reproduce this - I will create an issue from this thread and get it fixed. Thanks for letting us know!
